src-db/database/sourcedata/OBUIAPP_PROCESS.xml

```diff
@@ -12,7 +37,7 @@
 * Dependency in JAR format: the process will simply delete the dependency register.
 * Dependency in Source format (downloaded from Module Management window or Local): apart from deleting the dependency, the process will also erase the dependency source folders from the environment.
 This process only deletes the selected module, not its dependencies.]]></HELP>
-<!--69839B215B1D4876BBD6723819627F7C-->  <ACCESSLEVEL><![CDATA[4]]></ACCESSLEVEL>
+<!--69839B215B1D4876BBD6723819627F7C-->  <ACCESSLEVEL><![CDATA[7]]></ACCESSLEVEL>
```

Comment:
Why was this change made?

Reply:
In this particular case, the access level change was generated with the export.database command

Reply:
> Why was this change made?

In this particular case, the access level change was generated with the export.database command

Reply:
Hmmm it's kind of strange. @Matias-Bernal do you know why would the export.database command change this process' access level?